tests/runtest.sh

```diff
@@ -382,11 +383,17 @@ function create_core_overlay {
 }
 
 function precompile_overlay_assemblies {
+    declare -A skipCrossGenFiles
+
+    while read line
+    do
+      skipCrossGenFiles[$line]=1
+    done <  "$(dirname "$0")/skipCrossGenFiles.$ARCH.txt"
 
     if [ $doCrossgen == 1 ]; then
         local overlayDir=$CORE_ROOT
 
-        filesToPrecompile=$(ls -trh $overlayDir/*.dll)
+        filesToPrecompile=$(find -L $overlayDir -iname \*.dll -not -iname \*.ni.dll -not -iname \*-ms-win-\* -type f )
```

Comment:
@wtgodbe Can you please file an issue against yourself to ensure that APISets are not part of the non-Windows overlay?